tests/integration/conftest.py

```diff
@@ -194,6 +194,22 @@ def compose_containers(
 
             yield docker_client
         finally:
+            # After tests finish, copy coverage data from container(s) to local machine
+            if coverage_file:
+                coverage_dir = str(Path(coverage_file).parent)
+                print(" * Copying coverage data back...")
+                for c in containers:
+                    if c.name in ("mongo_container",):
+                        continue
+                    flist = c.execute(
+                        ["ls", "-a", "/home/jobflow/coverage/"]
+                    ).splitlines()
+                    for file in flist:
+                        if file.startswith(".coverage"):
+                            c.copy_from(
+                                f"/home/jobflow/coverage/{file}",
+                                f"{coverage_dir}/{file}",
+                            )
```

Comment:
Even though it does not guarantee that everything works, maybe I should add a check that at least one file was copied ?

Reply:
I will not do that for now as I am not completely sure what do in case pytest is explicitly executed on a specific test file in which containers are not used. In that case it wouldn't copy a coverage file obviously ...